robottelo/constants.py

```diff
@@ -2,6 +2,13 @@
 """Defines various constants"""
 from nailgun import entities
 
+from robottelo.config import settings
+
+if not settings.configured:
+    settings.configure()
+
+FIXTURES_URL = settings.fixtures_url
```

Comment:
fixtures_url sounds a bit vague, I'd change the name to make it clearer what it represents, something like repo_hosting_url?

Reply:
Changed to `repos_hosting_url`, hope that makes sense!